velox/runner/Runner.h

```diff
@@ -28,10 +27,18 @@ namespace facebook::velox::runner {
 /// scan.
 class SplitSource {
  public:
+  /// Result of getSplits. Each split belongs to a group. A nullptr split for
+  /// group n means that there are on more splits for the group. In ungrouped
+  /// execution, the group is always 0.
```

Comment:
ungrouped execution is -1 or 0?
In Prestissimo, it is defined as max for ungrouped id:
```
uint32_t kUngroupedGroupId{std::numeric_limits<uint32_t>::max()
```